scene/main/http_request.h

```diff
@@ -51,6 +51,7 @@ class HTTPRequest : public Node {
 		RESULT_SSL_HANDSHAKE_ERROR,
 		RESULT_NO_RESPONSE,
 		RESULT_BODY_SIZE_LIMIT_EXCEEDED,
+		RESULT_BODY_DECOMPRESS_FAILED,
```

Comment:
You should also bind it in `_bind_methods`.

Reply:
@akien-mga thank you! Updated the PR.